src/test-helpers/typeFactory.ts

```diff
@@ -81,6 +81,34 @@ export class TypeFactory {
 		return key.setMeta(storageEntry.creator.meta);
 	}
 
+	/**
+	 * Similar to the storageKey method above, this is specific to storageKeys
+	 * that take in double map for keys.
+	 *
+	 * @param index The id to assign the key to.
+	 * @param indexTypeOne The First InterfaceType that will be used to create the index into its new appropriate index type
+	 * @param indexTypeOne The Second InterfaceType that will be used to create the index into its new appropriate index type
+	 * @param storageEntry Used primarily on QueryableStorageEntry (ie: api.query) within the polkadot api library.
+	 * Contains necessary key value pairs to retrieve specific information from a query
+	 * such as `at`, `entriesAt`, `entries` etc..
+	 *
+	 * Some Parameter Examples:
+	 * 1. apiPromise.query.assets.account
+	 */
+	storageKeyDoubleMap(
+		index: number,
+		indexTypeOne: keyof InterfaceTypes,
+		indexTypeTwo: keyof InterfaceTypes,
+		storageEntry: StorageEntryBase<'promise', GenericStorageEntryFunction>
+	): StorageKey {
+		const idOne = this.#registry.createType(indexTypeOne, index);
+		const idTwo = this.#registry.createType(indexTypeTwo);
+
+		const key = new StorageKey(this.#registry, storageEntry.key(idOne, idTwo));
```

Comment:
```suggestion
		const keys = new StorageKey(this.#registry, storageEntry.key(idOne, idTwo));
```
Maybe? Confusing though since its `StorageKey` singular

Reply:
Yea I think a singluar `key` here makes sense because we are instantiating a single StorageKey object.